eventsourcing/src/main/scala/com/evolutiongaming/akkaeffect/eventsourcing/JournalKeeper.scala

```diff
@@ -217,10 +230,11 @@ object JournalKeeper {
     }
 
     for {
-      deletedTo <- Ref[F].of(none[SeqNr])
-      timestamp <- Clock[F].millis
-      check      = Check(timestamp)
-      save       = Save(check, deletedTo)
+      deletedTo    <- Ref[F].of(none[SeqNr])
+      timestamp    <- Clock[F].millis
+      check         = Check(timestamp)
+      save          = Save(check, deletedTo)
+      snapshotOffer = snapshotOffer0.filter(_.persisted)
```

Comment:
Here's the only logical change: ignore non-persisted snapshot because it cannot be used in any way within snapshot persisting logic